layouts/jobs/list.html

```diff
@@ -5,149 +5,110 @@
+          throw new Error(`HTTP error! status: ${response.status}`);
+        }
+        const apiData = await response.json();
+        
+        // Process and transform the API response
+        const today = new Date();
+        today.setHours(0, 0, 0, 0);
+        
+        const filteredJobs = apiData.filter(job => {
+          // Filter jobs that are not past today
+          const archiveDate = new Date(job.archive_date);
+          archiveDate.setHours(0, 0, 0, 0);
+          return archiveDate >= today;
+        });
+
+        // Transform jobs to our format
+        const transformedJobs = filteredJobs.map(job => ({
+          title: job.title?.rendered || 'No title available',
+          app_deadline: job.archive_date,
+          job_url: extractJobUrl(job),
+          employer: extractEmployer(job)
+        }));
+
+        return transformedJobs;
+      } catch (error) {
+        console.error('Error fetching jobs:', error);
+        return []; // Return empty array if API fails
+      }
     };
     
-    function addJobs(dataArray) {
-      var jobsList = [], jobTypes = [], employer = [];
-      // Not insterested if the deadline have already passed
+    async function addJobs(dataArray) {
+      var jobsList = [], employer = new Set();
+      
+      // Filter and collect data in a single pass
+      const today = new Date();
+      today.setHours(0, 0, 0, 0);
       dataArray.forEach((job) => {
-        if (new Date(job.app_deadline) >= new Date().setHours(0,0,0,0)){
+        if (new Date(job.app_deadline) >= today){
           jobsList.push(job);
-          job.type.forEach((jt) => {if(!jobTypes.includes(jt.toLowerCase())){jobTypes.push(jt.toLowerCase())}});
-          if(!employer.includes(job.employer)){employer.push(job.employer);}
+          employer.add(job.employer);
         }
       });
+      
       // Sort the arrays
-      jobsList = jobsList.sort((a,b) => (a.app_deadline > b.app_deadline) ? 1 : (a.app_deadline < b.app_deadline) ? -1 : 0);
-      jobTypes = jobTypes.sort();
-      employer = employer.sort();
+      jobsList.sort((a,b) => (a.app_deadline > b.app_deadline) ? 1 : (a.app_deadline < b.app_deadline) ? -1 : 0);
+      const sortedEmployer = Array.from(employer).sort();
       
       // If no jobs to show, display appropriate message
       if (jobsList.length === 0) {
         var jobsHTML = `<div class="alert alert-secondary mt-4">
-                          We are currently unaware of any upcoming jobs or project opportunities relevant for data-driven 
-                          life science. If you know of any relevant jobs or project opportunities that should be shown 
-                          here, please suggest them to us using the "Tell us about an open position" button above.
+                          We are currently unaware of any upcoming job opportunities at the SciLifeLab Data Centre. 
+                          All positions shown here are specifically from the Data Centre team.
```

Comment:
I think the text can just say _"Currently there are no job opportunities at the SciLifeLab Data Centre"_

This site will represent the information from DC, so won't be good to say we are unaware of ourself 😄